docs/users/subscriptions/faq/index.md

```diff
@@ -83,7 +86,10 @@ For detailed instructions, see:
 ## What happens when I run out of credits?
 If you run out of credits you can keep using Devopness to manage your environment resources and all features on your paid plans will remain accessible.
 
-However, new actions will not be able to be created in your projects until you add credits to your account. See How do I add credits to my account?
+However, new actions will not be able to be created in your projects until you add credits to your account. 
```

Comment:
I believe the user will `not be able`, therefore `actions will not be created` or `actions cannot be created` could be easier to read than `actions will not be able to be created`?

Reply:
Agree. I was just correcting and standardizing the links, but I'll correct this text as well. 

I was thinking about later reviewing all the texts in the documentation too, not only to find grammar errors, but to see if they are following this writing pattern.